lib/install/utils.go

```diff
@@ -396,9 +425,12 @@ func tryInstallBinary(targetPath string, uid, gid int, logger log.FieldLogger) e
 	if err != nil {
 		return trace.Wrap(err, "failed to determine path to binary")
 	}
-	err = os.MkdirAll(filepath.Dir(targetPath), defaults.SharedDirMask)
-	if err != nil {
-		return trace.Wrap(err)
+	dir := filepath.Dir(targetPath)
+	if dir != string(filepath.Separator) {
```

Comment:
`os.MkdirAll("/")` requires additional permissions in a restricted environment :-)